### ISSUE TITLE
Using removed Babel 5 option: base.stage

### ISSUE THREAD

**Issue by alexicum** [gaearon/disposables#6]
https://github.com/halt-hammerzeit/webpack-react-redux-isomorphic-render-example/issues/21#issuecomment-284310365
`ReferenceError: [BABEL] [Projects]\webpack-react-redux-isomorphic-render-example\node_modules\disposables\modules\index.js: Using removed Babel 5 option: base.stage - Check out the corresponding stage-x presets http://babeljs.io/docs/plugins/#presets...`

Is it possible to add .babelrc to the .npmignore and release a new version?

**Comment by gaearon** [comment]
You should not be trying to compile anything in `node_modules` unless you really know what you’re doing. The problem isn’t with this library, it’s with your build setup.

**Comment by gaearon** [comment]
That said I published a "fix" in `disposables@1.0.2`.

**Comment by jasonfb** [comment]
this happened to me after I inadvertently added `stage: 1` to my `server/.babelrc` file. I don't see any disposables package in my json file but perhaps I am missing something.
```
{
  "presets": ["es2015", "stage-0"],
  "plugins": ["transform-runtime"],
   stage: 1
}
```

fixed by removing this

```
{
  "presets": ["es2015", "stage-0"],
  "plugins": ["transform-runtime"],
}
```
